kubernetes/linux/main.sh

```diff
@@ -1073,19 +1073,32 @@ if [ -e "/opt/dcr_env_var" ]; then
       setGlobalEnvVar LOGS_AND_EVENTS_ONLY "${LOGS_AND_EVENTS_ONLY}"
 fi
 
-setGlobalEnvVar AZMON_RESOURCE_OPTIMIZATION_ENABLED "${AZMON_RESOURCE_OPTIMIZATION_ENABLED}"
+setGlobalEnvVar ENABLE_CUSTOM_METRICS "${ENABLE_CUSTOM_METRICS}"
+if [ "${ENABLE_CUSTOM_METRICS}" == "true" ]; then
```

Comment:
Does this mean if someone has custom metrics on, they will not get the other benefits as well?

Reply:
yes, because resource optimization assumes that custom metrics is not needed. so if cx has enabled custom metrics we cannot enable resource optimization